docs/development/guidelines.md

```diff
@@ -370,7 +429,43 @@ Examples
 
 ## Tests
 
-If a function contains more code than just the getting or setting of a value, automated test should be added to the [`tests`][tests-folder] folder. The file structure in the tests folder should mirror the file structure of the [`src`][src-folder] folder.
+If a function contains more code
+than just the getting or setting of a value,
+automated test should be added
+to the [`tests`][tests-folder] folder.
```

Comment:
A little outdated now. Maybe instead write something like

> We aim for 100% line coverage, so for any new function automated tests ...

Reply:
Changed it.